test/parallel/test-tls-delayed-attach-error.js

```diff
@@ -0,0 +1,50 @@
+var common = require('../common');
+var assert = require('assert');
+
+if (!common.hasCrypto) {
+  console.log('1..0 # Skipped: missing crypto');
+  process.exit();
+}
+var tls = require('tls');
+
+var fs = require('fs');
+var net = require('net');
+
+var bonkers = new Buffer(1024);
+bonkers.fill(42);
+
+var receivedError = false;
+var ended = 0;
+
+var options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem'),
+  cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
+};
+
+var server = net.createServer(function(c) {
+  setTimeout(function() {
+    var s = new tls.TLSSocket(c, {
+      isServer: true,
+      secureContext: tls.createSecureContext(options)
+    });
+
+    s.on('_tlsError', function() {
+      receivedError = true;
+    });
+
+    s.on('close', function() {
+      ended++;
```

Comment:
That seems right. I was mostly just copying `test-tls-delayed-attach.js` which did this.